frontend/src/components/organisms/Following.tsx

```diff
@@ -43,29 +49,43 @@ export default function Following({ users }: Props) {
   return (
     <section className="section">
       <h2 className="section-title">팔로잉</h2>
-      {users.length ? (
-        <UserList
-          users={users}
-          imageSize={52}
-          hasStatus={true}
-          buttons={[
-            <GameInviteButton key="button0" />,
-            <Button onClick={handleClickDm} key="button1" primary size="small">
-              메시지 보내기
-            </Button>,
-            <Button
-              key="button2"
-              onClick={handleClickUnfollow}
-              linkStyle
-              className="text-red"
-              size="small"
-            >
-              언팔로우
-            </Button>,
-          ]}
-        />
+      {isFetching && !data ? (
+        <Spinner className="flex justify-center mt-2 mb-8" />
+      ) : users.length ? (
+        <InfiniteScroll
+          next={fetchNextPage}
+          hasMore={hasMore}
+          dataLength={users.length}
+          loader={<Spinner className="flex justify-center pt-2 pb-8" />}
+        >
+          <UserList
+            users={users}
+            imageSize={52}
+            hasStatus={true}
+            buttons={[
+              <GameInviteButton key="button0" />,
+              <Button
+                onClick={handleClickDm}
+                key="button1"
+                primary
+                size="small"
+              >
+                메시지 보내기
+              </Button>,
+              <Button
+                key="button2"
+                onClick={handleClickUnfollow}
+                linkStyle
+                className="text-red"
+                size="small"
+              >
```

Comment:
```suggestion
              <GameInviteButton key="gameInviteButton" />,
              <Button
                onClick={handleClickDm}
                key="dmButton"
                primary
                size="small"
              >
                메시지 보내기
              </Button>,
              <Button
                key="unfollowButton"
                onClick={handleClickUnfollow}
                linkStyle
                className="text-red"
                size="small"
              >
```

혹시 button key가 숫자였던 이유가 있었나요? 기억이 안남,,

Reply:
그냥 key 값이 필요해서 임의로 넣어두고 신경 안 쓴 거 같슴다